src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx

```diff
@@ -89,8 +89,10 @@ SwapModalHeaderProps) {
   const [showInverted, setShowInverted] = useState<boolean>(false)
 
   // show fiatValue for unadjusted trade amounts!
-  const fiatValueInput = useUSDCValue(trade.inputAmountWithoutFee)
-  const fiatValueOutput = useUSDCValue(trade.outputAmountWithoutFee)
+  // const fiatValueInput = useUSDCValue(trade.inputAmountWithoutFee)
+  // const fiatValueOutput = useUSDCValue(trade.outputAmountWithoutFee)
+  const fiatValueInput = useHigherUSDValue(trade.inputAmountWithoutFee)
```

Comment:
higher or highest?

Reply:
well technically "highest" would mean just that, the "highest" whereas "higher" implies of the choices given, which is "highest" 

but yes it's very nitpicky here. happy to change it to whatever we like best @alfetopito @anxo @nenadV91

Reply:
whatever